client/src/components/Tickets/TicketModal.js

```diff
@@ -170,6 +187,9 @@ const TicketHistoryCard = ({
       </CardHeader>
       <Collapse isOpen={isOpen}>
         <CardBody>{content}</CardBody>
+        <CardBody>
+          <Attachments attachments={attachmentTitles} />
+        </CardBody>
```

Comment:
We only want to render the `<Attachments>` component if `attachmentTitles` is not an empty array.
```suggestion
        <CardBody>
          <Attachments attachments={attachmentTitles} />
        </CardBody>
```